improver/utilities/mathematical_operations.py

```diff
@@ -356,3 +356,60 @@ def process(self, cube):
         integrated_cube = self.perform_integration(upper_bounds_cube, lower_bounds_cube)
 
         return integrated_cube
+
+
+def fast_linear_fit(x, y, axis=None, keepdims=False, gradient_only=False):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Argument name "y" doesn't conform to '[a-z_][a-z0-9_]{2,30}$' pattern](https://app.codacy.com/manual/metoppv_tech/improver/pullRequest?prid=5319684)